generators/app/index.js

```diff
@@ -242,6 +246,7 @@ module.exports = class extends Generator {
    */
   prompting() {
     if (this.options.skipPrompt) {
+      _.assign(this._preconfigs, this._getPromptDefaults());
```

Comment:
Previously we did not set defaults for `--hurry` so everything was just undefined.